crates/uv/src/commands/project/run.rs

```diff
@@ -63,7 +66,11 @@ pub(crate) async fn run(
     // Determine whether the command to execute is a PEP 723 script.
     let script_interpreter = if let RunCommand::Python(target, _) = &command {
         if let Some(metadata) = uv_scripts::read_pep723_metadata(&target).await? {
-            debug!("Found PEP 723 script at: {}", target.display());
+            writeln!(
+                printer.stderr(),
+                "Reading inline script metadata from: {}",
```

Comment:
A bit of a nit, but we've already read it right? And the user passed us this script name already? Should we just say "Using inline script metadata for requirements" (a little unsure about "requirements")

Reply:
I would prefer "Using inline script metadata for: {}"